home/views.py

```diff
@@ -16,37 +16,37 @@
 MEETUP_EVENTS_URL = 'https://api.meetup.com/Perth-Django-Users-Group/events/'
 
 
-def get_events(event_status, from_date):
-    try:
-        client = meetup.api.Client('73c42797541a6c207a2a2b41262a66')
-
-        group_info = client.GetGroup({'urlname': 'Perth-Django-Users-Group'})
-        try:
-            group_events = client.GetEvents({'group_id': group_info.id, 'status': event_status}).results
-        except ValueError:
-            group_events = []
-
-        iterable = (
-            (lambda event_datetime: {
-                'group_id': group_info.id,
-                'event_id': event['id'],
-                'event_name': event['name'],
-                'event_url': event['event_url'],
-                'og_event_name': '({}) {}'.format(dateformat.format(event_datetime, 'D d M'), event['name']),
-                'event_address': '{}, {}'.format(event['venue']['name'], event['venue']['address_1']) if 'venue' in event else '',
-                'event_description': event['description'],
-                'og_event_description': strip_tags(event['description']).encode('ascii', 'ignore'),
-                'event_yes_rsvp_count': event['yes_rsvp_count'],
-                'event_datetime': event_datetime,
-            })(datetime.datetime.fromtimestamp(event['time'] / 1000.0, PERTH_TIMEZONE))
-            for event in sorted(group_events, key=lambda d: d['time']))
-        return [
-            event
-            for event in iterable
-            if event['event_datetime'] >= from_date
-        ]
-    except HttpClientError:
-        return []
+# def get_events(event_status, from_date):
```

Comment:
just commented this out because it's no longer in use, not sure i want to remove it just yet because it might be useful for reference if further bugs show up... yeah we could look at previous commits, but... yeah...